src/Components/Form.css

```diff
@@ -0,0 +1,52 @@
+.add-new-book-section .add-book {
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.form .title {
+  width: 35%;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  padding: 3%;
```

Comment:
- [ ] Kindly reduce the padding around your input it should stick to the original design. check her the difference :point_down: 

Your inputs |
-----------|
![image](https://user-images.githubusercontent.com/65068771/155845628-07688153-477e-4097-8999-9ab276dba39e.png) |
The template inputs 
![image](https://user-images.githubusercontent.com/65068771/155845649-47ba407a-aab7-4cbf-9df9-976fd10484e9.png)